cmd/openfga/main_test.go

```diff
@@ -561,158 +560,6 @@ func GRPCCheckTest(t *testing.T, tester OpenFGATester) {
 				errorCode: codes.InvalidArgument,
 			},
 		},
-		{
-			name: "union_with_direct_check_single_operand",
-			input: &openfgapb.CheckRequest{
-				StoreId: ulid.Make().String(),
-				// AuthorizationModelId is generated automatically during testData bootstrap
-				TupleKey: tuple.NewTupleKey("repo:expressjs/express", "admin", "github|bob@openfga.com"),
```

Comment:
If we're going to delete the tests that have `Trace: true` then let's please add the trace resolution expectation to our `yaml` test suite 👍

Reply:
Added trace to the test suite. Added trace to a few of the tests. Thanks!